content/python/concepts/built-in-functions/terms/sum/sum.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: 'sum()'
+Description: 'Takes in an iterable object, such as a list or tuple, and returns the sum of all elements.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Functions'
+  - 'Iterators'
+  - 'Lists'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+---
+
+Takes in an iterable object, such as a list or tuple, and returns the sum of all elements.
+
+## Syntax
+
+```python
+sum(iterable, start=0)
+```
+
+## Example 1
```

Comment:
For all the examples can we replace the comment with the output with a ` ```shell ` block illustrating the ouput. For example, after this first one have:

````
This example results in the following output:

```shell
15
```
````

Rather than a comment in the code.

Reply:
True, the comment seems a bit odd this is a much better solution!